src/Sulu/Component/Content/Document/Subscriber/RouteSubscriber.php

```diff
@@ -43,5 +45,13 @@ public function handleMetadataLoad(MetadataLoadEvent $event)
             'property' => self::DOCUMENT_TARGET_FIELD,
             'type' => 'reference',
         ]);
+        $metadata->addFieldMapping('locale', [
+            'encoding' => 'system',
+            'property' => self::DOCUMENT_LOCALE_FIELD,
+        ]);
+        $metadata->addFieldMapping('history', [
+            'encoding' => 'system',
+            'property' => self::DOCUMENT_HISTORY_FIELD,
+        ]);
```

Comment:
This behavior will probably change in https://github.com/sulu-io/sulu/pull/2057.